walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js

```diff
@@ -19,8 +19,8 @@
 function randomizeImage() {
   // The images directory contains 13 images, so generate a random index between
```

Comment:
Update this comment now that you have 15 images

Reply:
Will do!